README.md

```diff
@@ -114,6 +116,13 @@ There are two types of configuration that `wrangler` uses: global user and per p
        When successful, this command will print out your user information, including the type of plan you
        are currently on.
 
+	- #### Using environment variables
+
+		You can also configure your global user with the following environment variables:
+
+	    - $CF_API_KEY for your Cloudflare API key
+	    - $CF_EMAIL for your Cloudflare account email
```

Comment:
you may want to give an example here:

```
CF_API_KEY=farts CF_EMAIL=a@farts.com wrangler publish --release
```

You may also mention that this is the preferred method for using wrnagler in CI!